testing/src/test/java/org/jdbi/v3/testing/junit5/JdbiGenericExtensionTest.java

```diff
@@ -37,13 +37,13 @@ public class JdbiGenericExtensionTest {
     @Test
     public void testApacheDerby() {
         List<String> userNames = derbyExtension.getJdbi().withHandle(h -> {
-                    try (Query query = h.createQuery("SELECT name FROM users ORDER BY id")) {
-                        return query.mapTo(String.class).list();
-                    }
-                });
+            try (Query query = h.createQuery("SELECT name FROM users ORDER BY id")) {
+                return query.mapTo(String.class).list();
+            }
+        });
 
-        assertThat(userNames).hasSize(2);
-        assertThat(userNames).containsExactly("Alice", "Bob");
+        assertThat(userNames).hasSize(2)
```

Comment:
these seem to be unrelated to the checkstyle changes.

Reply:
Dir entgeht auch nichts oder? 😄
this was flagged as 'code smell' not by Checkstyle but by the Github workflow